sdk/monitor/Azure.Monitor.Ingestion/tests/MonitorIngestionLiveTest.cs

```diff
@@ -192,5 +192,34 @@ public async Task ValidInputFromObjectAsJsonNoBatchingAsync()
             Assert.AreEqual(204, response.Status);
             Assert.IsFalse(response.IsError);
         }
+
+        [Test]
+        public async Task ValidInputGzippedNoBatchingAsync()
```

Comment:
Can we also add a test where the protocol method is called without a Gzipped payload, and the content-encoding is left blank?